book/website/community-handbook/content.md

```diff
@@ -0,0 +1,139 @@
+(ch-content)=
+# Writing Content
+
+As _The Turing Way_ project grows and evolves, we actively welcome and encourage continuous contributions from community members in the form of new chapters and case studies. 
+However, these new contributions need to be consistent with the overall theme, purpose, format, and style of the book's existing content.
+
+Since _The Turing Way_ is written asynchronously by multiple authors around the world, we have created several templates to guide future content contributions in order to preserve the book's consistency as it grows.
+
+This chapter provides a brief overview of these templates. 
+You can find each one in the `template` [directory](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/templates) of the _The Turing Way's_ Github repository and we encourage you to use them if you wish to contribute to content to _The Turing Way_. 
```

Comment:
I would maybe start each subsection by referring to the appropriate template. Then it becomes more of a workflow where it's like _so you want to start a new chapter?_ Alright, then go pick up your template from so-and-so folder (include a little description of the template, what it contains..just a bit) and then explain where to place the chapter file in the repo.